src/test/java/controllers/UserControllerTest.java

```diff
@@ -0,0 +1,4 @@
+package controllers;
```

Comment:
Don't include this file, UserControllerTest already exists under perkpack/controllers/UserControllerTest